ISSUE_TEMPLATE.md

```diff
@@ -1,39 +1,60 @@
-Please try to adhere to the guidelines below as much as possible when
-submitting your issue.
-- Verify that your issue is not being currently addressed by other
-[issues](https://github.com/scikit-learn/scikit-learn/issues?q=)
-or [pull requests](https://github.com/scikit-learn/scikit-learn/pulls?q=).
-- If your issue is a usage question or does not potentially require
-changes to the codebase to be solved, then
-[StackOverflow](http://stackoverflow.com/questions/tagged/scikit-learn)
-(using the`[scikit-learn]` tag) or our
-[mailing list](https://lists.sourceforge.net/lists/listinfo/scikit-learn-general)
-may be a better place to bring it up. For more information, see
-[User Questions](http://scikit-learn.org/stable/support.html#user-questions).
-
-If you are submitting a bug issue:
-- Please include your operating system type and version number, as well
-as your Python, scikit-learn, numpy, and scipy versions. This information
-can be found by runnning the following code snippet:
+<!--
```

Comment:
they're HTML comments, but they also work in markdown. Markdown doesn't have an official "commenting" scheme, but some users have found [some hacky solutions](http://stackoverflow.com/questions/4823468/comments-in-markdown). The downside of using `<!-- -->` for commenting is that it will still appear is the raw HTML, it just isn't displayed. That isn't quite relevant to this application, though, so I figured the ease of reading HTML comments would do.
If you look at the [rendered PR template](https://github.com/nelson-liu/scikit-learn/blob/refactor_templates/PULL_REQUEST_TEMPLATE.md) and the [rendered issue template](https://github.com/nelson-liu/scikit-learn/blob/refactor_templates/ISSUE_TEMPLATE.md), you will see that the comments do not appear.